core/state_processor.go

```diff
@@ -403,6 +403,7 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 	for i, tx := range block.Transactions() {
 		if isPoSA {
 			if isSystemTx, err := posa.IsSystemTransaction(tx, block.Header()); err != nil {
+				bloomProcessors.Close()
```

Comment:
`defer bloomProcessors.Close()` is more appropriate.

Reply:
[4f3cc9b](https://github.com/antimatter-dao/bas-template-bsc/pull/9/commits/4f3cc9b63bfaacef7472a9d762f367b4d6780768) ok

Reply:
👍🏻 LGTM